### PR TITLE
Rchanda1392 graphconnectors

### PR DESCRIPTION
**Pull request by rchanda1392**

### CODE REVIEW

File: concepts/custom-connector-sdk-overview.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ ms.prod: search
 description: "Use the Microsoft Graph connectors SDK to create custom Microsoft Graph connectors."
 ---
 
-# Microsoft Graph connectors SDK overview
+# Microsoft Graph connectors SDK (preview) overview
```
Comment (Lauragra) [inline concepts/custom-connector-sdk-overview.md:11]: If we're calling this a preview SDK, we should refer to it as such throughout, i.e. in mentions in all topics. We should also add (preview) to the TOC entry, "Use the connectors SDK".

Comment (rchanda1392) [reply concepts/custom-connector-sdk-overview.md:11]: Hi @Lauragra can we include our feedback email address (MicrosoftGraphConnectorsFeedback@service.microsoft.com) for customers to reach out for troubleshooting?

Comment (Lauragra) [reply concepts/custom-connector-sdk-overview.md:11]: > Hi @Lauragra can we include our feedback email address ([MicrosoftGraphConnectorsFeedback@service.microsoft.com](mailto:MicrosoftGraphConnectorsFeedback@service.microsoft.com)) in the docs for customers to reach out for troubleshooting?

You can, but I would avoid exposing the email address in the topic, instead just have link text like [Microsoft Graph Connectors team](mailto:MicrosoftGraphConnectorsFeedback@service.microsoft.com).

File: concepts/custom-connector-sdk-contracts-connectorcrawler.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ ms.prod: search
 description: "Learn about the contracts connector crawler API and models in the Microsoft Graph connectors SDK."
 ---
 
-# Microsoft Graph connectors SDK contracts connector crawler API and models
+# Microsoft Graph connectors SDK (preview) contracts connector crawler API and models
 
 The Microsoft Graph connectors SDK contracts connector crawler API and models are called during a crawl.
```
Comment (Lauragra) [inline concepts/custom-connector-sdk-contracts-connectorcrawler.md:13]: Global: Please make sure that all mentions of the SDK in the titles and in the article text include the preview, for clarity/consistency.

**Comment by rchanda1392** [comment]
Hi @Lauragra I've addressed the review comments, please do review.

**Comment by rchanda1392** [comment]
@Lauragra can you please help in reviewing and closing this PR?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**